rpi5.yaml

```diff
@@ -120,12 +120,19 @@ components:
         - "../meta-xt-rpi5-prod-devel"
 
       target_images:
+        - "tmp/deploy/images/%{MACHINE}/u-boot-%{MACHINE}.bin"
+        - "tmp/deploy/images/%{MACHINE}/boot.scr"
         - "tmp/deploy/images/%{MACHINE}/xen-%{MACHINE}"
         - "tmp/deploy/images/%{MACHINE}/xenpolicy-%{MACHINE}"
         - "tmp/deploy/images/%{MACHINE}/%{XT_XEN_DTBO_NAME}"
         - "tmp/deploy/images/%{MACHINE}/%{XT_DOMD_DTB_NAME}"
+        - "tmp/deploy/images/%{MACHINE}/%{SOC_FAMILY}-%{MACHINE}-mmc.dtbo"
+        - "tmp/deploy/images/%{MACHINE}/%{SOC_FAMILY}-%{MACHINE}-usb.dtbo"
+        - "tmp/deploy/images/%{MACHINE}/mmc-passthrough.dtbo"
+        - "tmp/deploy/images/%{MACHINE}/usb-passthrough.dtbo"
         - "tmp/deploy/images/%{MACHINE}/bcm2712-rpi-5-b.dtb"
         - "tmp/deploy/images/%{MACHINE}/%{XT_DOMD_IMAGE}-%{MACHINE}.rootfs.ext4"
+        - "tmp/deploy/images/%{MACHINE}/Image.gz"
```

Comment:
You already have 
```
        - "tmp/deploy/images/%{MACHINE}/Image.gz-%{MACHINE}.bin"
```

Why do you need `Image.gz`?

Reply:
Okay, so you deleted 

```
        - "tmp/deploy/images/%{MACHINE}/Image-%{MACHINE}.bin"
``` 

and added 

```
        - "tmp/deploy/images/%{MACHINE}/Image.gz"
```

Could you please explain why? And why in this particular commit? How it is related to switching between MMC and USB?

Reply:
Rouge uses only Image.gz for creating image

Reply:
You are doing this in the commit
```
rpi5.yaml: Add domd root device switch

Add DOMD_ROOT parameter. This parameter switches domd root device
to mmc or usb.
When usb option chopsen rootfs partition is not created in full.img.
Instead rootfs.img with rootfs partition can be created.
```

I agree,this is the correct change. But commits should be atomic, remember? Please move this change to a separate commit. In this commit leave only changes that are related to "Add domd root device switch"

Reply:
Moved this to separate commit